src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -0,0 +1,160 @@
+          <Button
+            data-testid={dataTestId.unpublishActions.openUnpublishModalButton}
+            variant="outlined"
+            onClick={() => setShowDeleteModal(true)}>
+            {t('common.delete')}
+          </Button>
+        </Breadcrumbs>
+      </Box>
+      <Modal
+        dataTestId="delete-registration-modal"
+        headingText={t('registration.delete_registration')}
+        open={showDeleteModal}
+        onClose={() => setShowDeleteModal(false)}>
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+          <Box>
+            <RequiredDescription />
+          </Box>
+          <Box sx={{ gap: '1rem' }}>
+            <Typography variant="h3">{t('unpublish_actions.unpublish_registration')}</Typography>
+            <Typography>{t('unpublish_actions.unpublish_registration_detail_1')}</Typography>
+            <Typography>{t('unpublish_actions.unpublish_registration_detail_2')}</Typography>
+          </Box>
+
+          <Formik
+            initialValues={{
+              deleteMessage: '',
+            }}
+            validationSchema={deleteValidationSchema}
+            onSubmit={(values, _formikHelpers) => {
+              handleDelete(values);
+            }}>
+            {({ values, errors, touched, handleChange, handleBlur, handleSubmit }) => (
+              <form onSubmit={handleSubmit}>
+                <Box sx={{ my: '1rem' }}>
+                  <Typography variant="h3">{t('registration.is_registration_error_question')}</Typography>
+                  <Typography gutterBottom={true}>{t('unpublish_actions.unpublish_registration_reason')}</Typography>
+                  <TextField
+                    variant="filled"
+                    fullWidth
+                    id="deleteMessageId"
+                    name="deleteMessage"
+                    data-testid={dataTestId.unpublishActions.unpublishJustificationTextField}
+                    label={t('common.justification')}
+                    placeholder={t('common.fill_textfield')}
+                    required={true}
+                    value={values.deleteMessage}
+                    onChange={handleChange}
+                    onBlur={handleBlur}
+                    error={touched.deleteMessage && Boolean(errors.deleteMessage)}
+                    helperText={touched.deleteMessage && errors.deleteMessage}
+                  />
+                </Box>
+
+                <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', my: '1rem' }}>
+                  <Box sx={{ gap: '1rem' }}>
+                    <Typography variant="h3">
+                      {t('unpublish_actions.unpublish_registration_duplicate_question')}
+                    </Typography>
+                    <Typography>
+                      {t('unpublish_actions.unpublish_registration_duplicate_citation_information')}
+                    </Typography>
+                  </Box>
+                </Box>
```

Comment:
her er det en del styling som motvirker hverandre og skaper kompleksistet uten noen effekt. God trening å gå gjennom egen PR og spørre seg selv hva man prøver å oppnå med hver linje kode, da det er enkelt å glemme å fjerne ting som ikke lenger er relevant etter senere endringer.

I dette tilfellet er det en `Box sx={{ gap: '1rem' }}`, men `gap` har ikke noen effekt med mindre det er en `grid` eller `flex`. Ytterst er det en flex (som skal brukes til å liste opp flere elementer), men den har bare en child, så da utgår den også. Det eneste som gjenstår da er `my`, så kan være at du ønsker å wrappe dette i en `<Box sx={{ my: '1rem' }}>` da.
Anyways: anbefaler å gå gjennom egen kode med lupe også, ingenting er bedre enn å fjerne kode uten å miste funksjonalitet 😎 

```suggestion
                <Typography variant="h3">
                  {t('unpublish_actions.unpublish_registration_duplicate_question')}
                </Typography>
                <Typography paragraph>
                  {t('unpublish_actions.unpublish_registration_duplicate_citation_information')}
                </Typography>
```